packages/astro/src/core/config/schema.ts

```diff
@@ -119,6 +120,7 @@ export const AstroConfigSchema = z.object({
 				.transform((val) => new URL(val)),
 			assets: z.string().optional().default(ASTRO_CONFIG_DEFAULTS.build.assets),
 			assetsPrefix: z.string().optional(),
+			assetsRemoveOriginals: z.boolean().default(ASTRO_CONFIG_DEFAULTS.build.assetsRemoveOriginals),
```

Comment:
I would add this under `image.` instead, `assets` here does not have the same connotation (it refers to all assets, like CSS and JS files)

Reply:
@Princesseuh, `image.removeOriginals` sounds good?

Reply:
Yep!